ethcore/src/state.rs

```diff
@@ -289,6 +284,25 @@ impl State {
 		}))
 	}
 
+	fn query_pod(&mut self, query: &PodState) {
+		query.get().iter().foreach(|(ref address, ref pod_account)| {
+			if self.get(address, true).is_some() {
```

Comment:
can it? `map` (and therefore `filter_map`) relies on there being return data, no? `foreach` allows there to be side-effects without return data.